src/ng/animate.js

```diff
@@ -233,6 +234,45 @@ var $AnimateProvider = ['$provide', /** @this */ function($provide) {
 
   /**
    * @ngdoc method
+   * @name $animateProvider#customFilter
+   *
+   * @description
```

Comment:
I see what you mean now. Good point. I will update the API docs and guide.
BTW, I think we should re-order the conditions, because now `customFilter` and `classNameFilter` are executed even if animations are globally disabled or the document is hidden etc.